.github/workflows/cd.yml

```diff
@@ -0,0 +1,40 @@
+name: CD
+
+on:
+    push:
+        branches:
+            - master
```

Comment:
Should we have something here or a bit down below that skips if it was a version bump?